content/posts/Tanmoy_Sarkar_Phase_1_Report.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Improve Development Experience: Phase 1 Report (GSoC 2023)"
+date: 2023-07-07T12:04:13+05:30
+draft: false
+author: Tanmoy Sarkar
+type: post
+---
+
+
+This blog is dedicated to my progress in the first phase of Google Summer of Code  2023.
+
+
+## About My Project
+
+My project focuses on making the process of working with CircuitVerse easier and more efficient. The main goal is to simplify setting up the development environment and working with the code. Additionally, Our aim to enhance the test cases and improve the continuous integration (CI) workflow, which will ultimately reduce the time required for code review by maintainers.
+
+In the first phase, we have covered
+
+
+
+* Integrate Ruby Debugger [Week 1]
+* Integrate Solargraph LSP [Week 2]
+* Integrate Vite Rails [Week 3]
+* RBS Installation & CI Integration [Week 3]
+* Integrate Undercover CI [Week 4]
+* Improve Unit testcase coverage [Week 5]
+* Improve integration testing [Week 6]
+
+
+### Integrate Ruby Debugger
+
+This integration enhances the debugging experience for developers by providing a visual debugging capability directly from their integrated development environment (IDE). It allows anyone to conveniently set breakpoints and debug the application from their IDE. This integration support  also docker based environment. By changing configuration in IDE , it can easily connect to TCP based debugger and debug the application easily. Documentation has been updated accordingly.
```

Comment:
```suggestion
This integration enhances the debugging experience for developers by providing a visual debugging capability directly from their integrated development environment (IDE). It allows anyone to conveniently set breakpoints and debug the application from their IDE. This integration also support docker based environment. By changing configuration in IDE , it can easily connect to TCP based debugger and debug the application easily. The documentation has been updated accordingly.
```

Could you also embed `Document` with respective doc link

Reply:
as the PR has not yet merged, should I embed the PR branch's documentation?

Reply:
No worries, we can add that after!